helms/tekton/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: tekton
+version: 0.1.0
+appVersion: v0.6.1
+description: Tekton chart
+keywords:
+  - cicd
+  - tekton
+maintainers:
+  - name: Alexey Kharlamov
```

Comment:
Is he still a Maintainer or it's a copy-paste?

Reply:
Removed